pkg/apis/core/types.go

```diff
@@ -2409,6 +2411,12 @@ const (
 	TolerationOpEqual  TolerationOperator = "Equal"
 )
 
+// PodReadinessGate contains the reference to a pod condition
```

Comment:
Not sure why the `add ReadinessGates in pod spec` commit is in this PR...

Reply:
Because it was not merged at the time...